assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -74,20 +76,27 @@ const IconWrapper = styled.div`
 // Since the svg-s are 32px and have extra room around, this needs to be removed
 const Collapse = styled.button`
   border: none;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
   background: transparent;
-  color: inherit;
+  color: ${({ theme, isCollapsed }) =>
+    isCollapsed ? theme.colors.fg.secondary : theme.colors.fg.primary};
   height: 32px;
   display: flex; /* removes implicit line-height padding from child element */
   padding: 0 4px 0 0;
   cursor: pointer;
+  margin-left: -12px;
+  transition: ${BUTTON_TRANSITION_TIMING};
+
+  &:hover,
+  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR}, &[data-focus-visible-added] {
```

Comment:
Oh we have a constant for this too!

```suggestion
  &.${ThemeGlobals.FOCUS_VISIBLE_SELECTOR}, &[${ThemeGlobals.FOCUS_VISIBLE_DATA_ATTRIBUTE}] {
```